decidim-comments/app/frontend/application/icon.component.jsx

```diff
@@ -2,20 +2,25 @@ import { PropTypes } from 'react';
 
 import assetUrl      from '../support/asset_url';
 
-const Icon = ({ name }) => {
+const Icon = ({ name, iconExtraClassName }) => {
```

Comment:
Well... I try to avoid `className` because it's already a real attribute name (jsx don't support `class`). Also it makes clear it will be a class for the icon svg element (or the `span` for phantom) so if the component has extra markup it's less confusing.